src/commands/test.ts

```diff
@@ -0,0 +1,134 @@
+import {Command} from './command';
+import * as wct from 'web-component-tester';
+// import * as process from 'process';
+
+export class TestCommand implements Command {
+  name = 'test';
+
+  description = 'Runs web-component-tester';
+
+  args = [
```

Comment:
Is there any way to get this from wct, or are we only going to try that for the newer batch of command-line-arg based tools?